LogIn/Auth/AuthManager.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  AuthManager.swift
+//  LogIn
+//
+//  Created by Aleksei Permiakov on 11.01.2022.
+//
+
+import FirebaseAuth
+import Combine
+
+final class AuthManager {
+    
+    static let shared = AuthManager()
+    private init() {}
+    
+    var isLoggedIn = CurrentValueSubject<Bool, Never>(false)
```

Comment:
Do not expose to the public writable interface. Anyone can change this right now

Reply:
ah yes